### DISCUSSION TITLE
How to download it

### DISCUSSION THREAD

**Discussion by Redactedslair** [Cvolton/GMDprivateServer#763] [Q&A] [answered]
I dont know how to download this as i wanna make a private server but this is stopping me.

**Comment by xtclovver** [comment, marked as answer]
![Screenshot_35](https://user-images.githubusercontent.com/44809071/119305125-fc4a0300-bc70-11eb-8e6e-4e7541cf4d00.png)
@Drtrayblox7855

**Comment by tzt345** [comment]
> I dont know how to download this as i wanna make a private server but this is stopping me.

If your on android make sure you go on desktop site & you will see a code button & press the download zip & your done!

**Comment by notqoid** [comment]
> I dont know how to download this as i wanna make a private server but this is stopping me.

if you're on mobile, 
![Screenshot_20210826_234548](https://user-images.githubusercontent.com/87503354/130994144-f94c190f-2aaf-47d6-b086-b5b378e055f9.jpg)

**Reply by Cvolton** [reply]
sus

**Reply by Czechball** [reply]
amogus

**Reply by Redactedslair** [reply]
Lol
